.github/workflows/cd-cli-npm.yml

```
@@ -1,4 +1,4 @@
-name: CD - Publish @common-grants/cli
+name: CD - Publish CLI Package (alpha)
```

```suggestion
name: CD - Deploy CLI Package to npm (alpha)
```

To match `CD - Deploy Core Package to npm`